HiP-DataStore/Controllers/RoutesController.cs

```diff
@@ -105,7 +195,10 @@ public IActionResult Get(RouteQueryArgs args)
                     .ToList();
 
                 if (invalidIds.Count > 0)
-                    return StatusCode(422, ErrorMessages.ExhibitNotFoundOrNotPublished(invalidIds[0]));
+                {
+                    response = StatusCode(422, ErrorMessages.ExhibitNotFoundOrNotPublished(invalidIds[0]));
```

Comment:
This issue and other things related to validation will be addressed in a separate PR, if that's ok.